distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -522,7 +522,10 @@ public void waitForHttpEndpoint(String path) throws InterruptedException {
 
     while (!available) {
       Response response = get(path);
-      available = response.getStatusCode() == 200 && response.getBody().print().length() > 0;
+      available = response.getStatusCode() == 200 && response.getBody().asString().length() > 0;
+      if (LOGGER.isDebugEnabled()) {
```

Comment:
To preempt questions about this check, adding this suppresses a somewhat valid sonarlint finding regarding the issue of passing parameters that are expensive to evaluate into a logger call.

Even though it's just "building a string", some of the strings corresponding to these response bodies are quite large.